metadata.txt

```diff
@@ -19,14 +19,25 @@ email=tim@kartoza.com
 # Optional items:
 
 # Uncomment the following line and add your changelog:
-# changelog=
+changelog=
+	v2.0.0:
+	- Code converted to Python 3
+	- Plugin can now be installed in QGIS 3
+	- Fixed a language translation issue
```

Comment:
@vermeulendivan did the translation issue exist on the past plugin version? I think this entry should be about the updates made on translation if there were any.